### ISSUE TITLE
Show Tags in alphabetical order 

### ISSUE THREAD

**Issue by howdytom** [keeweb/keeweb#1907]
**Describe the bug**
Tags do not show in random order in the left sidebar panel.

**To Reproduce**
Creating one Tag to various entries will display Tags in random order.

**Expected behavior**
By default Tags should show in alphabetical order in the left sidebar panel.

**Comment by antelle** [comment]
Makes sense, now they show in the order they occur in entries, which makes them harder to read and find.
